### ISSUE TITLE
No results when querying Wikidata

### ISSUE THREAD

**Issue by kvistgaard** [zazuko/vscode-sparql-notebook#5]
To reproduce:

This query
```sparql
PREFIX wd: <http://www.wikidata.org/entity/>
PREFIX wdt: <http://www.wikidata.org/prop/direct/>
PREFIX wikibase: <http://wikiba.se/ontology#>
PREFIX bd: <http://www.bigdata.com/rdf#>

SELECT DISTINCT ?ms ?msLabel ?hos ?hosLabel 
 
WHERE {
  ?ms wdt:P463 wd:Q458;
      wdt:P35 ?hos .
      SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
}
```
gives results both on Wikidata and when run from an external client, for example from [here](https://api.triplydb.com/s/8mvYGRETC).

The same query gives no output (not even empty set or error) from executed from SPARQL Notebook.

**Comment by kinghuang** [comment]
I'm having a similar problem. Queries seem to run correctly and I get a green checkmark in the cell, but the results are not displayed.

**Comment by BenjaminHofstetter** [comment]
@kvistgaard the problem here is the charset appended to the content type.  'application[/sparql-results]()+json;charset=utf-8'

**Comment by BenjaminHofstetter** [comment]
@kinghuang @kvistgaard  this should be resolved with 0.0.9 please update the extension and try again.

**Comment by BenjaminHofstetter** [comment]
Fixed in 0.0.9
